sessions/GraphQL-100/102/client/src/components/OrdersPage.js

```diff
@@ -17,7 +17,7 @@ const OrdersPage = props => (
     <Query query={GET_ORDERS}>
       {({ loading, error, data }) => {
         if (loading) return <p>Loading...</p>;
-        if (error) return <p>Oops something went wrong 😵</p>;
+        if (error) return <p>Oops something went wrong</p>;
         console.log(data.getOrders);
         return data.getOrders.map(order => (
```

Comment:
This bit wasn't working for me locally I got it to work with 
```suggestion
        return data.getOrders.map(order => {
          if (!order) return <p>Oops something went wrong</p>;
          return <div key={order.id} className="orderRow">
            <p>Order ID: {order.id}</p>
            <p>{moment(parseInt(order.orderDate)).format("HH:mm D/M/YY")}</p>
            <p>Product: {order.product.name}</p>
          </div>
        });
```

Reply:
It works fine for me, your suggestion didn't work?